interface/super/edit_layout.php

```diff
@@ -565,7 +565,7 @@ class='optin noselect' style='width:100%' />";
 
         echo "<input type='text' name='fld[$fld_line_no][list_id]'  id='fld[$fld_line_no][list_id]' value='" .
         htmlspecialchars($linedata['list_id'], ENT_QUOTES) . "'".$type.
-        " size='6' maxlength='30' class='optin listid' style='width:100%;cursor:pointer'".
+        " size='6' maxlength='100' class='optin listid' style='width:100%;cursor:pointer'".
```

Comment:
I see what you're saying, if we're allowing `some_really_long_string_text_here_for_the_list_id` that could be up to 100 characters long in the db then the input would require the same length. Disregard above comment.